### DISCUSSION TITLE
Show and tell

### DISCUSSION THREAD

**Discussion by amandaghassaei** [amandaghassaei/glcompute#5] [Show and tell]
I'll kick this thread off with an awesome project by @esperanc – calculating fractals by repeatedly inverting points with a set of circles, all in an interactive Observable notebook:

https://observablehq.com/@esperanc/circle-inversion-fractals

![circle inversion fractal - claudio esperanca](https://user-images.githubusercontent.com/3218981/214392311-f0493551-4820-499c-9927-24032a860efa.gif)

Also check out Claudio's port of the gpu-io examples to Observable: https://observablehq.com/@esperanc/gpu-io

<img width="672" alt="Screen Shot 2023-01-24 at 11 47 06 AM" src="https://user-images.githubusercontent.com/3218981/214393537-167c2c64-7be2-41be-b9e0-027fbbda7adf.png">

**Comment by esperanc** [comment]
Cool! Thanks

--Claudio

**Comment by esperanc** [comment]
Extra cool!

--Claudio


Em seg., 3 de ago. de 2026 às 18:35, Forrest O. ***@***.***>
escreveu:
